custodia/secrets.py

```diff
@@ -31,9 +33,11 @@ def _db_container_key(self, default, trail):
         if len(trail) > 1:
             f = self._db_key(trail)
         elif len(trail) == 1 and trail[0] != '':
+            self.logger.debug('Forbidden action: Wrong container path. '
+                              'Container names must end with "/".')
```

Comment:
Custodia uses quoting the other way around ```"foo 'whatever' bar"```. Also I prefer double quotes for human readable output because it makes it easier to include single quotes: ```"It's good."```.